pkg/api/metrics.go

```diff
@@ -19,6 +20,7 @@ type metrics struct {
 	RequestCount     prometheus.Counter
 	ResponseDuration prometheus.Histogram
 	PingRequestCount prometheus.Counter
+	ErrorPageViews   *prometheus.CounterVec
```

Comment:
I would consider having a breakdown of all status code groups - 1xx 2xx 3xx 4xx 5xx

Reply:
Just mentioning our discussion here for record.

We have decided to do the grouping on graphana side instead. It seems to be better to not have the extra lookups on the client-side.